lib/handlers/subscribe.js

```diff
@@ -159,12 +187,15 @@ function addSubs (sub, done) {
     func = blockDollarSignTopics(func)
   }
 
+  /* c8 ignore start */
   if (client.closed || client.broker.closed) {
     // a hack, sometimes client.close() or broker.close() happened
     // before authenticate() comes back
     // we don't continue subscription here
+    // since it is a rare race condition we ignore it in coverage testing
     return
   }
+  /* c8 ignore stop */
```

Comment:
was this tested before?

Reply:
Not that I know, the coverage testing shows that this part is never reached and given the comments it looked like a rare race condition to me.